### PR TITLE
demo: update openssl-qat-engine image

### PR DESCRIPTION
**Pull request by mythi**
Closes: #893

### CODE REVIEW

File: demo/openssl-qat-engine/Dockerfile [hunk 1]
```diff
@@ -1,14 +1,26 @@
-ARG FINAL_BASE_IMAGE=clearlinux:base
-
-FROM clearlinux:base as builder
+FROM ubuntu:21.10 as builder
```
Comment (bart0sh) [inline demo/openssl-qat-engine/Dockerfile:1]: Any reason to use ubuntu 21.10 for this demo? Can we use LTS (20.04/focal)?

Comment (mythi) [reply demo/openssl-qat-engine/Dockerfile:1]: no particular reason. My thinking was "as new as possible" but I'll move to `focal`

**Review by bart0sh — APPROVED** [review]
/lgtm

**Comment by mythi** [comment]
the build was stuck, re-pushed

**Review by bart0sh — DISMISSED** [review]
/lgtm

**Pull request merged**